### PR TITLE
clean: Clarify the repository configuration permission DOCS-658

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
Clarify the repository configuration permissions.

### :eyes: Live preview
https://docs-658-clarify-permission--docs-codacy.netlify.app/organizations/roles-and-permissions-for-organizations/

### CODE REVIEW

File: docs/organizations/roles-and-permissions-for-organizations.md [hunk 1]
```diff
@@ -92,7 +92,25 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
...
+      <td>Configure repository quality gates and goals</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Enable repository local analysis, manage repository API tokens</td>
```
[the remark this answers is not in the document]
Comment (claudiacarpinteiro) [reply docs/organizations/roles-and-permissions-for-organizations.md]: @nicklem I incorporated your feedback for the new operations description.

File: docs/organizations/roles-and-permissions-for-organizations.md [hunk 2]
```diff
@@ -92,7 +92,25 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
...
+      <td>Configure repository quality gates and goals</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Configure repository to run analysis on local build server, manage repository API tokens</td>
```
Comment (nicklem) [inline docs/organizations/roles-and-permissions-for-organizations.md:113]: Now that I'm reading it, I'm not convinced anymore 😅
We can either revert to your initial suggestion or reword this slightly. Up to you.

```suggestion
      <td>Set repository analysis to run on a local build server, manage repository API tokens</td>
```

File: docs/organizations/roles-and-permissions-for-organizations.md [hunk 3]
```diff
@@ -92,7 +92,25 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
...
+      <td colspan="2" class="no">No</td>
+      <td class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Configure repository quality gates and goals</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td colspan="2" class="no">No</td>
```
Comment (mrfyda) [inline docs/organizations/roles-and-permissions-for-organizations.md:107]:
```suggestion
      <td colspan="2" class="yes">Yes</td>
```

File: docs/organizations/roles-and-permissions-for-organizations.md [hunk 4]
```diff
@@ -228,7 +246,25 @@ The table below maps the GitLab Cloud and GitLab Enterprise roles to the corresp
...
+      <td class="no">No</td>
+      <td colspan="2" class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td colspan="2" class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Configure repository quality gates and goals</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td class="no">No</td>
```
Comment (mrfyda) [inline docs/organizations/roles-and-permissions-for-organizations.md:261]:
```suggestion
<td class="no">No</td>
```

Comment (mrfyda) [inline docs/organizations/roles-and-permissions-for-organizations.md:261]:
```suggestion
      <td class="yes">Yes</td>
```

**Review by mrfyda — APPROVED** [review]

File: docs/organizations/roles-and-permissions-for-organizations.md (hunk 2 above)
Comment (claudiacarpinteiro) [reply docs/organizations/roles-and-permissions-for-organizations.md:113]: Using **Enable** instead, to match the description on the UI.

```suggestion
      <td>Enable repository analysis to run on a local build server, manage repository API tokens</td>
```

**Comment by claudiacarpinteiro** [comment]
@mrfyda As confirmed, I updated the quality gates and goals permission according to the expected behavior, Admin.
Can you please have a final look? 🙏

**Comment by mrfyda** [comment]
> @mrfyda As confirmed, I updated the quality gates and goals permission according to the expected behavior, Admin. Can you please have a final look? 🙏

all good! 👌

**Pull request merged**